src/OutputFormatter/GraphVizOutputFormatter.php

```diff
@@ -87,7 +87,11 @@ public function finish(
         }
 
         if ($dumpDotPath = (string) $outputFormatterInput->getOption(self::DUMP_DOT)) {
-            file_put_contents($dumpDotPath, (string) $graph);
+            $dotContent = (string) $graph;
+            // make sure we get a uniform line-ending in the dotfile
+            $dotContent = str_replace("\r\n", "\n", $dotContent);
+            file_put_contents($dumpDotPath, $dotContent);
```

Comment:
before this change the generated file had a mix of `\n` and `\r\n` line endings.. not sure where the root cause is